lib/factory_logger/engine.rb

```diff
@@ -1,3 +1,5 @@
+require_relative './config.rb'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.